.circleci/config.yml

```diff
@@ -225,7 +225,7 @@ jobs:
           command: sudo pip install awscli
       - run:
           name: Deploy to S3
-          command: aws s3 sync ~/project/docs/6.9.6 s3://purchases-docs/android/6.9.6 --delete
+          command: aws s3 sync ~/project/docs/6.9.7 s3://purchases-docs/android/6.9.7 --delete
       - run:
           name: Update index.html
           command: aws s3 cp ~/project/docs/index.html s3://purchases-docs/android/index.html
```

Comment:
Hmm I just noticed but would this make the 6.9.7 docs the default ones? If so, we would need to either fix it so we don't overwrite the file if the versions is older than the one already there, or overwrite this file after publishing this

Reply:
Yup! We can remove that for now 👍